test/mock/kernel/mock_framework.zig

```diff
@@ -5,6 +5,8 @@ const expectEqual = std.testing.expectEqual;
 const GlobalAllocator = std.debug.global_allocator;
 const TailQueue = std.TailQueue;
 const warn = std.debug.warn;
+const builtin = @import("builtin");
+const Modifier = builtin.CallOptions.Modifier;
```

Comment:
Is this needed as can't find it in the code

Reply:
It certainly isn't. It remains from when I was looking into replacing @noInlineCall